vtex-apps/breadcrumb
# Update CSS handles and add new props

#### What is the purpose of this pull request?

- Use new `vtex.css-handles` app
- Add new CSS handles: `homeLink`, `termArrow`
- Add variations to already existing handles, `link` and `arrow` so that the user can customize each arrow and each link individually.
- Add new props: `homeIconSize` and `caretIconSize` that enables the user to control de `size` property passed to icons used by `BaseBreadcrumb`.

#### How should this be manually tested?

Take a look at [Workspace](https://breadcrumb--storecomponents.myvtex.com/apparel---accessories/shoes/mizuno?map=c,c,b) and inspect the breadcrumb to check the CSS classes being applied.

#### Screenshots or example usage

<img width="1021" alt="Screen Shot 2019-11-04 at 10 23 45" src="https://user-images.githubusercontent.com/27777263/68124138-61060700-feed-11e9-9656-1b59a4c3bf92.png">


<img width="1150" alt="Screen Shot 2019-11-04 at 10 24 31" src="https://user-images.githubusercontent.com/27777263/68124146-66fbe800-feed-11e9-971c-6817356d1d23.png">

#### Types of changes
